student/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+################
+# Docker file to build a docker servlet container hosting the Student WAR.
+################
+
+#Based off of the shared secure tomcat 7 image from SS_SharedMultiJar
+FROM fwsbac/secure-tomcat-image:3.0.1
```

Comment:
We're using our secure tomcat7 image from SS_SharedMultiJar which has been published to DockerHub.